frontend/packages/pipelines-plugin/src/components/repository/repository-form-utils.ts

```
@@ -77,39 +79,41 @@ export const pipelinesAccessTokenValidationSchema = (t: TFunction) =>
-        then: yup.object().shape({
-          user: yup
-            .string()
-            .matches(nameRegex, {
-              message: t(
-                'pipelines-plugin~Name must consist of lower-case letters, numbers and hyphens. It must start with a letter and end with a letter or number.',
-              ),
-              excludeEmptyString: true,
-            })
-            .required(t('pipelines-plugin~Required')),
-        }),
+        then: (schema) =>
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)